crates/burn-import/src/onnx/dim_inference.rs

```diff
@@ -122,6 +123,24 @@ fn constant_update_outputs(node: &mut Node) {
     };
 }
 
+fn constant_of_shape_update_output(node: &mut Node) {
+    let value_type = node
+        .attrs
+        .get("value")
+        .map(|v| v.clone().into_tensor().elem_type)
+        .unwrap_or(ElementType::Float32); // If not given, defaults to 0 as float32
+
+    if let ArgType::Tensor(input_type) = &node.inputs[0].ty {
+        node.outputs[0].ty = ArgType::Tensor(TensorType {
+            elem_type: value_type,
+            dim: input_type.dim,
+            shape: None,
+        });
+    } else {
```

Comment:
Again, I think we should make sure that the input is a 1D tensor.

Reply:
Fixed it by introducing shape type.